src/main/java/org/fundacionjala/sfdc/pageobjects/opportunities/SFONewModifyPage.java

```diff
@@ -1,4 +1,4 @@
-package org.fundacionjala.sfdc.pageobjects.opportunities;
+package org.fundacionjala.sfdc.pageobjects.Opportunities;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "SFONewModifyPage.java" should be located in "org/fundacionjala/sfdc/pageobjects/Opportunities" directory, not in "/home/travis/build/AT-06/sfdc-tests/src/main/java/org/fundacionjala/sfdc/pageobjects/opportunities". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)